README.md

```diff
@@ -387,7 +387,9 @@ if (process.env.NODE_ENV === 'production') {
 // Then in your <head>
 {css}
 ```
+### CSS modules
 
+[Css modules](https://github.com/css-modules/css-modules) are support by default for all files with `.module.(css|less|scss|sass)` extension.
```

Comment:
Perhaps add examples of two css files where one (main.css) *wouldn't* use CSS Modules and another (`my-awesome-component.module.css`) would be.